.github/workflows/extension_ci.yml

```diff
@@ -9,9 +9,13 @@ on:
   pull_request:
     branches:
       - main
+    paths-ignore:
+      - 'pgmq-rs/**'
   push:
     branches:
       - main
+    paths-ignore:
+      - 'pgmq-rs/**'
```

Comment:
the postgres extension CI should not run when there are changes to the client SDKs